deps/+wxWidgets/wxWidgets.patch

```diff
@@ -0,0 +1,15 @@
+--- /x/CMakeLists.txt	2024-07-17 21:35:52.584239800 -0600
++++ ./build/cmake/lib/webview/CMakeLists.txt	2024-07-17 21:35:15.566885700 -0600
+@@ -15,7 +15,11 @@
+     if(CMAKE_SIZEOF_VOID_P EQUAL 4)
+         set(WEBVIEW2_ARCH x86)
+     else()
+-        set(WEBVIEW2_ARCH x64)
++        if (CMAKE_SYSTEM_PROCESSOR MATCHES "^(ARM64|aarch64)$")
++            set(WEBVIEW2_ARCH arm64)
++        else()
++            set(WEBVIEW2_ARCH x64)
++        endif()
+     endif()
+ 
+     add_custom_command(TARGET ${target} POST_BUILD
```

Comment:
The branches diverged. But this might not be necessary anymore.

Reply:
I merged this patch into the existing webview.patch file.